modules/nodes-pods-vertical-autoscaler-about.adoc

```diff
@@ -11,6 +11,11 @@ The VPA automatically computes historic and current CPU and memory usage for the
 
 The VPA automatically deletes any pods that are out of alignment with its recommendations one at a time, so that your applications can continue to serve requests with no downtime. The workload objects then re-deploy the pods with the original resource limits and requests. The VPA uses a mutating admission webhook to update the pods with optimized resource limits and requests before the pods are admitted to a node. If you do not want the VPA to delete pods, you can view the VPA resource limits and requests and manually update the pods as needed.
 
+[NOTE]
+====
+By default, workload objects must specify a minimum of two replicas in order for the VPA to automatically delete their pods. Workload objects that specify fewer replicas than this minimum are not deleted. If you manually delete these pods, when the workload object redeploys the pods, the VPA does update the new pods with its recommendations. You can change this minimum by modifying the `VerticalPodAutoscalerController` object as shown shown in _About changing the VPA minimum value_.
```

Comment:
The two `[NOTE]`s mention the section title as _About changing the VPA minimum value_, but it looks like the title is actually just _Changing the VPA minimum value_.